infrastructure/tooling/src/publish/tasks.js

```diff
@@ -24,15 +25,16 @@ module.exports = ({tasks, cmdOptions, credentials, baseDir, logsDir}) => {
   ensureTask(tasks, {
     title: 'Get release version',
     requires: [],
-    provides: ['release-version'],
+    provides: ['release-version', 'docker-flow-version'],
     run: async (requirements, utils) => {
       if (cmdOptions.staging) {
         return {
           'release-version': '9999.99.99',
+          'release-revision': '999999999',
```

Comment:
I think this should be `docker-flow-version` but also be a JSON'd object

Reply:
Good point!